package.json

```diff
@@ -47,6 +48,13 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
     "prebuild": "prebuildify -t 8.14.0 -t electron@3.0.0 --napi --strip",
+    "prebuild:cross:android-arm": "prebuildify-cross --target android-arm",
+    "prebuild:cross:android-arm64": "prebuildify-cross --target android-arm64",
+    "prebuild:cross:linux-armv6": "prebuildify-cross --target linux-armv6",
+    "prebuild:cross:linux-armv7": "prebuildify-cross --target linux-armv7",
+    "prebuild:cross:linux-arm64": "prebuildify-cross --target linux-arm64",
+    "prebuild:cross:linux-x64": "prebuildify-cross --target linux-x64",
+    "prebuild:cross:windows-x64": "prebuildify-cross --target windows-x64",
```

Comment:
At the moment we use `AppVeyor` for running tests on windows and also running `prebuildify` (maybe switching to Travis for windows a bit later) so I think we want to keep that as is for now. Also, the same goes for osx. But very cool that this is so multi platform from the start, e.g. including windows and mac.

Reply:
yes, that's all good, i haven't tested windows so no idea if it works at all and [`dockcross` doesn't support mac](https://github.com/dockcross/dockcross/issues/163#issuecomment-331728940).

Reply:
Might be interesting to look into later; it's possible that mingw builds are faster (in build time or runtime) or smaller.